vislcg3-to-conllu.py

```diff
@@ -189,7 +189,12 @@ def trykk(buffer, tokcount, charcount, t): #{
 			lem = rbase.findall(llinia)[0];	
 			categs = rcateg.findall(llinia);	
 			mor = rparent.findall(llinia)[0];
-			etiqueta = rfunc.findall(llinia)[0].strip();
+			etiqueta = rfunc.findall(llinia)
+			if not etiqueta:  # {
+				etiqueta = 'X';
```

Comment:
Would it be possible to make it `dep` instead of X ? And maybe leave a note in the MISC Column? `Missing=Dep`

Reply:
Should it be `dep`for both a main reading and sub-reading? You were assigning `X` to main readings missing a label, so I'd just followed that.